src/widgets/company/Rules/styled.ts

```diff
@@ -0,0 +1,21 @@
+import { styled } from 'styled-components';
+
+import { StHeading3 } from '@/shared/styles/global';
+
+export const StRules = styled.div`
+  max-width: 800px;
+  padding: 80px;
+`;
+export const StRulesWrap = styled.div``;
+export const StRulesTitle = styled(StHeading3)`
+  font-size: 1rem;
+  margin-bottom: 40px;
+`;
+export const StRulesList = styled.ul`
+  list-style-type: decimal;
+  margin-left: 24px;
```

Comment:
Используй `list-style-position` вместо `margin`, тогда нумерация будет позиционироваться внутри списка
```suggestion
  list-style-position: inside;
```

Reply:
тогда будет не по макету
![Screenshot 2023-08-14 121340](https://github.com/agricultural-machinery-rental/client/assets/63591058/75864a46-38e4-4fc5-b4ec-ab46ef047549)

Reply:
Да, согласна. Я не учла отступы на всех строках.
Ты сделал отлично!